src/app/home-page/widget/filtered-list.ts

```diff
@@ -0,0 +1,28 @@
+import { OnInit } from '@angular/core';
+import { Observable, Subject } from 'rxjs';
+import { UserQuery } from 'app/shared/user/user.query';
+import { DefaultService } from 'app/shared/openapi';
+
+/**
+ * Base class for logged in homepage widgets that have a filter list
+ */
+export abstract class FilteredList implements OnInit {
+  userId$: Observable<number>;
+  protected ngUnsubscribe: Subject<{}> = new Subject();
```

Comment:
You need to call ngUnsubscribe.next and ngUnsubscribe.complete. Or just let Base do it for you.